app/services/bulk_api_import/importer.rb

```diff
@@ -0,0 +1,30 @@
+class BulkApiImport::Importer
+  def initialize(resource_list:)
+    @resources = resource_list
+  end
+
+  def import
+    @resources.each do |resource|
+      resource_importer(resource).import if import_enabled?(resource)
+    end
+  end
+
+  # TODO:
+  # This method is a stopgap to ensure that our mock imports and real imports can coexist.
+  # Once we have implemented all resource types, we should remove this method entirely.
+  def import_enabled?(resource)
+    %w[Patient].include?(resource[:resourceType])
+  end
```

Comment:
```suggestion
    IMPORTERS[resource[:resourceType]].present?
  end
```

Reply:
I'll skip this one, because this stopgap method will go away in the next PR.